cmd/router/main.go

```diff
@@ -183,7 +194,8 @@ func graphHandler(w http.ResponseWriter, req *http.Request) {
 }
 
 var (
-	jsonGraph = flag.String("graph-json", "", "serialized json graph def")
+	jsonGraph          = flag.String("graph-json", "", "serialized json graph def")
+	headersToPropagate = strings.Split(os.Getenv("PROPAGATE_HEADERS"), ",")
```

Comment:
Where do you specify the environment variable?

Reply:
No particular preference here. I just followed what we discussed over slack chat.